### ISSUE TITLE
Proposal: add CRUD api to add/remove/modify streams

### ISSUE THREAD

**Issue by kottle** [AlexxIT/go2rtc#232]
It could be useful have api CRUP to add remove streams at runtime without restart go2rtc every time the configuration changes.

If you think it worth with this feature, I can implement it

Thanks for your work.

**Comment by NickM-27** [comment]
There is already an API to add and remove streams.

**Comment by kottle** [comment]
That's right, I did few tests with DELETE api/streams and PUT and I noticed config file doesn't chage with new configuration so after a reboot you lose changes.

It's an expected behaviour or is it a bug ?
Thanks

**Comment by NickM-27** [comment]
I believe that is expected behavior, and honestly the preferred behavior. There is a separate API to edit the config file.

**Comment by kottle** [comment]
This is my suggestion to handle it https://github.com/AlexxIT/go2rtc/pull/233
